element_lab/lab.py

```diff
@@ -86,9 +90,31 @@ class Project(dj.Lookup):
     definition = """
     project                 : varchar(32)
     ---
-    project_description=''         : varchar(1024)
+    project_description=''  : varchar(1024)
     """
 
+    class Keywords(dj.Part):
+        definition = """
+        # Project keywords, exported dataset meta info
+        -> master
+        keyword: varchar(32)
+        """
+
+    class Publication(dj.Part):
+        definition = """
+        # Project's resulting publications
+        -> master
+        publication: varchar(256)
+        """
+
+    class SourceCode(dj.Part):
+        definition = """
+        -> master
+        repository_url     : varchar(256) # URL to source code for replication
+        ---
+        repository_name='' : varchar(32)
+        """
+
```

Comment:
Should these tables be `Manual` tables so that data in these tables can be added at different times?  See notes below from the [DataJoint Docs](https://docs.datajoint.org/python/computation/03-master-part.html#populating).
```
Note that the entities in the master and the matching entities in the part are inserted within a single make call of the master, which means that they are a processed inside a single transactions: either all are inserted and committed or the entire transaction is rolled back. This ensures that partial results never appear in the database.
```


In which case, we will need to update the `<table>_to_nwb_dict` functions.

Reply:
That makes sense, thanks. I've updated both the schema and the export functions accordingly